docs/content/guide/component.ngdoc

```diff
@@ -109,13 +109,16 @@ change will be reflected in the parent component. For components however, only t
 the data should modify it, to make it easy to reason about what data is changed, and when. For that reason,
 components should follow a few simple conventions:
 
-  - Inputs are realized with `@` and `=` bindings
+  - Inputs are realized with `<` bindings. The `<` symbol denotes {@link $compile#-scope- one-way bindings} which are
```

Comment:
My concern was really that `@` should be used when you are most likely not to pass an expression but a simple string. By changing it from `@` to `<` (or `=`) your attribute must change from `<editable-field field-type="text">` to `<editable-field field-type="'text'">` (note the extra quotes).